Dockerfile

```diff
@@ -1,9 +1,6 @@
 FROM ubuntu:latest
 
-MAINTAINER Kenneth Daily <kenneth.daily@sagebase.org>
+MAINTAINER Tess Thyer <tess.thyer@sagebase.org>
```

Comment:
Was reluctant to make this change but Kenny's not going to be maintaining this anymore